event.go

```diff
@@ -229,9 +229,13 @@ func (s *Session) onInterface(i interface{}) {
 	case *GuildUpdate:
 		setGuildIds(t.Guild)
 	case *VoiceServerUpdate:
-		go s.onVoiceServerUpdate(t)
+		if !s.DisableVoiceHandlers {
```

Comment:
If we're going to rename and change purpose of the option then we also need to change it here
```suggestion
		if s.HandleVoiceGateway {
```